### DISCUSSION TITLE
Mobile phone open link error

### DISCUSSION THREAD

**Discussion by mnz12138** [rainbow-me/rainbowkit#1257] [Q&A] [answered]
Open the link in your mobile browser rainbow://wc?uri=wc:174d989fba570b61d4f0a9aa2831852e9e2997d145f3e1087aaa8aca9ced87d5@2?symKey=ff746707dad4e401e461421e774d79bd301c3a7494f0434bffba5160c01e1858&relay-protocol=irn
![IMG_0013](https://github.com/rainbow-me/rainbowkit/assets/13588391/278b9ab0-a0e5-477c-bf45-73197ab9cc66)

**Comment by DanielSinclair** [comment, marked as answer]
WalletConnect v2 is more stable in recent versions of Wagmi since you experienced this, but if you are still seeing this issue, please open an Issue on the repo and we can further investigate.
